public/discussion.php

```diff
@@ -1,8 +1,8 @@
+<?php require_once (dirname(__FILE__)."/../modules/ensure_logged_in.php"); ?>
+
 <?php
 require_once "../modules/models/discussion.php";
-require_once "../common.php";
-
-ensure_logged_in();
+require_once (dirname(__FILE__)."/./modules/ensure_logged_in.php");
```

Comment:
Might be wrong, but is this included twice in the same file? Is there a reason for that?

Reply:
Whoops you're right